libkbfs/bcache.go

```diff
@@ -214,8 +218,10 @@ func (b *BlockCacheStandard) makeRoomForSize(size uint64) bool {
 // PutWithPrefetch implements the BlockCache interface for BlockCacheStandard.
 func (b *BlockCacheStandard) PutWithPrefetch(
 	ptr BlockPointer, tlf tlf.ID, block Block, lifetime BlockCacheLifetime,
-	hasPrefetched bool) error {
+	hasPrefetched bool) (err error) {
 
+	_, cacheGetError := b.Get(ptr)
```

Comment:
Would we ever call this twice on a permanent block?  Maybe this check should go below in each of the lifetime cases, checking the respective cache for that lifetime.

Also it's inherently racy since there's no lock taken across the get and the put.  That's probably not worth fixing, but could use a TODO here.